shared/AppInsightsCore/src/JavaScriptSDK/SenderPostManager.ts

```diff
@@ -93,16 +93,17 @@ export class SenderPostManager {
                     _disableFetchKeepAlive = !!config.disableFetchKeepAlive;
     
                     _fallbackInst = { sendPOST: _xhrSender} as IXHROverride;
-                    if (!_isOneDs) {
-                        _sendCredentials = false; // for appInsights, set it to false always
+                    // remove since 3.3.1
+                    // if (!_isOneDs) {
+                    //     _sendCredentials = false; // for appInsights, set it to false always
+                    // }
+                    let location = getLocation();
+                    if (location && location.protocol && location.protocol.toLowerCase() === "file:") {
+                        // Special case where a local html file fails with a CORS error on Chromium browsers
+                        _sendCredentials = false;
```

Comment:
not quite sure about this part.   Previously, when disableCredentials is true, and the code is not being run from a local HTML file, then sendCredentials will still be ture. Is this by design?